packages/react-router-dev/vite/route-chunks-test.ts

```diff
@@ -178,13 +178,58 @@ describe("route chunks", () => {
         "import { shared } from "./shared";
         export const chunk = shared("chunk");"
       `);
-      expect(getChunkedExport(code, "main", {}, ...cache)?.code)
+      expect(omitChunkedExports(code, ["chunk"], {}, ...cache)?.code)
         .toMatchInlineSnapshot(`
         "import { shared } from "./shared";
```

Comment:
This is the core issue being fixed. In the original test case using `getChunkedExport`, this import statement was missing.